profiling-ffi/src/profiles.rs

```diff
@@ -438,6 +480,43 @@ pub unsafe extern "C" fn ddog_prof_Profile_add_endpoint_count(
     profile.add_endpoint_count(endpoint, value);
 }
 
+/// Add an upscaling rule which will be use to adjust values and make them
+/// closer to reality.
+///
+/// # Arguments
+/// * `profile` - a reference to the profile that will contain the samples.
+/// * `offset_values` - offset of the values
+/// * `label_name` - name of the label used to identify sample(s)
+/// * `label_value` - value of the label used to identify sample(s)
+/// * `ratio` - value to multiply the sample values at `offset_values` against
+///
+/// # Safety
+/// This function must be called before serialize and must not be called after.
+/// The `profile` ptr must point to a valid Profile object created by this
+/// module.
+/// This call is _NOT_ thread-safe.
+#[must_use]
+#[no_mangle]
+pub unsafe extern "C" fn ddog_prof_Profile_add_upscaling_rule(
```

Comment:
To be honest, I did not see any "how to use API" documentation for the other, so that's why I did not think about adding.
In addition to that, there is the example where you could see how to use it.
Maybe we should reconsider this for all the FFI function.

Reply:
Yeah. The example in this case is a quite simple use of the feature (no-op scaling, no labels, only a single index "0") which is also why I couldn't get a lot from it.